core/domain/exp_domain_test.py

```diff
@@ -3859,6 +4257,7 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
     content_ids_to_audio_translations:
       content: {}
       default_outcome: {}
+      new_content: {}
```

Comment:
Why this change?

Reply:
I wanted to test [this line](https://codecov.io/gh/oppia/oppia/src/develop/core/domain/exp_domain.py#L2138). For this I need to create `extra_content_ids_in_citat` and this is possible if I add new_content